src/new/wallet.js

```diff
@@ -184,6 +184,129 @@ class HathorWallet extends EventEmitter {
     return historyTransactions;
   }
 
+  /**
+   * Get utxos of the wallet addresses
+   *
+   * @param {object} options Consolidation options shared with filterUtxos and consolidateUtxos (https://github.com/HathorNetwork/hathor-wallet-headless/issues/44)
+   *
+   * @return {object} { total_amount_available: number, total_utxos_available: number, total_amount_locked: number, total_utxos_locked: number, utxos: Array[] } utxos and meta information about it
```

Comment:
Can we format it with multiple lines? It is hard to read the way it is.

Reply:
Done